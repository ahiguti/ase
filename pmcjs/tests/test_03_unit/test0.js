
use('Perl');
print("args:");
for (var i in arguments) {
  print(arguments[i]);
}
var pargv = Perl.Arrays[''].ARGV;
print("perl args:");
for (var i in pargv) {
  print(pargv[i]);
}
