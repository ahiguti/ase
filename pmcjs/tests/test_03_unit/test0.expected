args:
foo
bar
baz
perl args:
foo
bar
baz
